pages/getPreviewApiKey/index.tsx

```diff
@@ -0,0 +1,24 @@
+import dynamic from "next/dynamic";
+import { FC, useEffect } from "react";
+
+import { webAuth } from "../../lib/constants/auth";
+
+const GetPreviewApiKey: FC = () => {
+  useEffect(() => {
+    const exitPreview = async () => {
+      await fetch('/api/exit-preview');
+    }
+    exitPreview();
```

Comment:
You do no wait for the promise because you do not await the promise returned from the `exitPreview` function. Why do you even create the function, it doesn't seem to be useful here.